packaging/linux/common/khiops_env.in

```diff
@@ -79,16 +82,29 @@ if [ -f "/usr/share/khiops/khiops.jar" ]; then
     fi
 fi
 
-@ADDITIONAL_EN_VAR@
+get_script_dir(){
+    SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+    echo "$SCRIPT_DIR/"
+}
```

Comment:
This function is not used in `khiops_env` and is redefined in the same way in `khiops` and `khiops_coclustering` which use it.